src/bundle/Resources/public/scss/_inputs.scss

```diff
@@ -5,32 +5,46 @@
 }
 
 .form-check-inline {
-    .ez-input {
-        &--checkbox {
-            position: relative;
-        }
+    .ibexa-input {
+        position: relative;
     }
 }
 
-.ez-input {
+.ibexa-input {
+    position: relative;
+    display: inline-block;
     outline: none;
-    transition: all 0.2s $ibexa-admin-transition;
+    transition: all $ibexa-admin-transition-duration $ibexa-admin-transition;
+    border: calculateRem(1px) solid $ibexa-color-dark-300;
+    background-color: $ibexa-white;
+
+    &:hover {
+        border-color: $ibexa-color-primary;
+        cursor: pointer;
+    }
+
+    &:active,
+    &:focus {
+        border-color: $ibexa-color-primary;
```

Comment:
Not all active inputs have this border colour. Actually, not sure if any? Checkboxes do not have `:active` state, do they?